tests/tools/console.test.ts

```diff
@@ -149,5 +155,40 @@ describe('console', () => {
         );
       });
     });
+
+    describe('issues type', () => {
+      beforeEach(() => {
+        setIssuesEnabled(true);
+      });
+      afterEach(() => {
+        setIssuesEnabled(false);
+      });
+
+      it('lists issues', async () => {
+        await withBrowser(async (response, context) => {
+          const page = await context.newPage();
+          const issuePromise = new Promise<void>(resolve => {
+            page.once('issue', () => {
+              resolve();
+            });
+          });
+          await page.setContent('<input type="text" name="username" />');
+          await issuePromise;
+          await listConsoleMessages.handler({params: {}}, response, context);
```

Comment:
we should not be re-using the same response object for multiple calls but I see that this already happened in existing tests. let's create a new response instance for the second call.